api/pom.xml

```diff
@@ -68,6 +68,12 @@
 			<version>${log4jVersion}</version>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+			<groupId>javax.servlet</groupId>
+			<artifactId>javax.servlet-api</artifactId>
+			<version>${javaxServletVersion}</version>
+			<scope>provided</scope>
```

Comment:
if only required for tests, then scope should test

Reply:
Angshu, 
scope of javax.servlet-api dependency changed to test

Reply:
Angshu,
Changed javax.servlet-api dependency scope to test